ruby/src/main/java/com/google/protobuf/jruby/RubyEnumDescriptor.java

```diff
@@ -0,0 +1,113 @@
+package com.google.protobuf.jruby;
+
+import com.google.protobuf.DescriptorProtos;
+import com.google.protobuf.Descriptors;
+import org.jruby.Ruby;
+import org.jruby.RubyClass;
+import org.jruby.RubyModule;
+import org.jruby.RubyObject;
+import org.jruby.RubyNumeric;
+import org.jruby.anno.JRubyClass;
+import org.jruby.anno.JRubyMethod;
+import org.jruby.runtime.Block;
+import org.jruby.runtime.ObjectAllocator;
+import org.jruby.runtime.ThreadContext;
+import org.jruby.runtime.builtin.IRubyObject;
+
+/**
+ * Created by isaiah on 15/12/14.
```

Comment:
The date here is in DD/MM/YY while the others are MM/DD/YY :-)

Can we actually remove these "Created by..." comments? We totally give you credit and appreciate the huge contribution here but we usually don't have individual authorship comments on new classes that we add.